docs/reference/objects/product/variant/index.md

```diff
@@ -221,6 +221,8 @@ An array containing the [prices]({% link docs/reference/objects/product/price.md
 - When accessed through an [experience slot]({% link docs/reference/objects/product/experience_slot.md %}), it will return the prices for the variant on the given slot.
 - When accessed independently or through a product, it will return the default prices for the variant, managed in **Experience > Variants**.
 
+*Note:* this returns up to 50 elements.
```

Comment:
```suggestion
*Note:* this returns up to 50 prices.
```

Reply:
Yeah wasn't sure if adding "prices" sounded weird but perhaps elements is too technical too, happy to change.